app/managers/websocket_manager.ts

```diff
@@ -270,7 +268,9 @@ class WebsocketManager {
             return;
         }
 
-        this.closeAll();
+        if (!this.netConnected) {
```

Comment:
Why the check?

Reply:
Good question. I was trying to be "careful" and the the previous if was leaving too many possibilities to handle. But thinking it through, the only scenario this should happen is the following:

- We are in the background
- The network changes from disconnected to connected

In that case the clients should already be closed, so we are just avoiding trying to close them again, which is not a big deal. I can remove the check.